src/yelp-application.c

```diff
@@ -117,6 +117,49 @@ struct _YelpApplicationPrivate {
 };
 
 static void
+open_uri (YelpApplication *app,
+          YelpUri         *uri,
+          gboolean         new_window,
+          gboolean         fallback_help_list);
+
+static void 
+activate_show_page (GSimpleAction *action,
+                    GVariant      *parameter,
+                    gpointer       user_data)
+{
+    YelpApplication *app = YELP_APPLICATION (user_data);
+    YelpUri *uri = yelp_uri_new (DEFAULT_URI);
```

Comment:
What about my earlier suggestion about exporting the `DEFAULT_URI` macro in yelp-uri.h as `YELP_GNOME_HELP_URI`?